CMakeLists.txt

```diff
@@ -0,0 +1,1808 @@
+  countryflags/US.png
+  countryflags/UY.png
+  countryflags/UZ.png
+  countryflags/VA.png
+  countryflags/VC.png
+  countryflags/VE.png
+  countryflags/VG.png
+  countryflags/VI.png
+  countryflags/VN.png
+  countryflags/VU.png
+  countryflags/WF.png
+  countryflags/WS.png
+  countryflags/XBZ.png
+  countryflags/XCA.png
+  countryflags/XEN.png
+  countryflags/XES.png
+  countryflags/XGA.png
+  countryflags/XNI.png
+  countryflags/XSC.png
+  countryflags/XWA.png
+  countryflags/YE.png
+  countryflags/ZA.png
+  countryflags/ZM.png
+  countryflags/ZW.png
+  countryflags/default.png
+  countryflags/index.json
+  deadtee.png
+  editor/automap/grass_doodads.json
+  editor/automap/grass_main.json
+  editor/background.png
+  editor/checker.png
+  editor/cursor.png
+  editor/desert_main.rules
+  editor/entities.png
+  editor/grass_main.rules
+  editor/jungle_main.rules
+  editor/winter_main.rules
+  emoticons.png
+  fonts/DejaVuSans.ttf
+  game.png
+  languages/belarusian.json
+  languages/bosnian.json
+  languages/brazilian_portuguese.json
+  languages/bulgarian.json
+  languages/chuvash.json
+  languages/czech.json
+  languages/danish.json
+  languages/dutch.json
+  languages/finnish.json
+  languages/french.json
+  languages/gaelic_scottish.json
+  languages/german.json
+  languages/greek.json
+  languages/hungarian.json
+  languages/index.json
+  languages/italian.json
+  languages/japanese.json
+  languages/korean.json
+  languages/kyrgyz.json
+  languages/license.txt
+  languages/norwegian.json
+  languages/polish.json
+  languages/portuguese.json
+  languages/readme.txt
+  languages/romanian.json
+  languages/russian.json
+  languages/serbian.json
+  languages/simplified_chinese.json
+  languages/slovak.json
+  languages/spanish.json
+  languages/swedish.json
+  languages/turkish.json
+  languages/ukrainian.json
+  mapres/bg_cloud1.png
+  mapres/bg_cloud2.png
+  mapres/bg_cloud3.png
+  mapres/desert_doodads.png
+  mapres/desert_main.png
+  mapres/desert_mountains.png
+  mapres/desert_mountains2.png
+  mapres/desert_sun.png
+  mapres/generic_deathtiles.png
+  mapres/generic_lamps.png
+  mapres/generic_shadows.png
+  mapres/generic_unhookable.png
+  mapres/grass_doodads.png
+  mapres/grass_main.png
+  mapres/jungle_background.png
+  mapres/jungle_deathtiles.png
+  mapres/jungle_doodads.png
+  mapres/jungle_main.png
+  mapres/jungle_midground.png
+  mapres/jungle_unhookables.png
+  mapres/light.png
+  mapres/moon.png
+  mapres/mountains.png
+  mapres/snow.png
+  mapres/stars.png
+  mapres/sun.png
+  mapres/winter_doodads.png
+  mapres/winter_main.png
+  mapres/winter_mountains.png
+  mapres/winter_mountains2.png
+  mapres/winter_mountains3.png
+  maps/ctf1.map
+  maps/ctf2.map
+  maps/ctf3.map
+  maps/ctf4.map
+  maps/ctf5.map
+  maps/ctf6.map
+  maps/ctf7.map
+  maps/ctf8.map
+  maps/dm1.map
+  maps/dm2.map
+  maps/dm3.map
+  maps/dm6.map
+  maps/dm7.map
+  maps/dm8.map
+  maps/dm9.map
+  maps/license.txt
+  maps/readme.txt
+  maps/sur1.map
```

Comment:
this should be `lms1.map`

Reply:
This PR is based on an older version of the source tree. I will update it once I get time to do it.

Reply:
why do you choose to list every files ? (instead of your first version on ddnet that used recursive glob)
I guess there is a reason but I can't figure it out

Reply:
> why do you choose to list every files ? (instead of your first version on ddnet that used recursive glob)
> I guess there is a reason but I can't figure it out

I'm curious about that too.

Reply:
https://stackoverflow.com/questions/32411963/why-is-cmake-file-glob-evil